packages/dart_frog_test/example/test/routes/index_test.dart

```diff
@@ -0,0 +1,21 @@
+import 'dart:io';
+
+import 'package:dart_frog_test/dart_frog_test.dart';
+import 'package:test/test.dart';
+
+import '../../routes/index.dart' as route;
+
+void main() {
+  test(
+    'responds with a 200 and "Welcome to Dart Frog!".',
+    () async {
+      final testContext = DartFrogTestContext(path: '/');
```

Comment:
how is the path relevant in this case?

Reply:
Tbh, it isn't, but to be able to make the `Uri`, we need a full path, so `Uri.parse` will work. I don't like the idea of assuming the path or just using a placeholder, so I figured it out we should always request the path in order to avoid confusions.